app/src/main/java/com/github/libretube/ui/preferences/BackupRestoreSettings.kt

```diff
@@ -77,18 +87,54 @@ class BackupRestoreSettings : BasePreferenceFragment() {
         }
     }
 
+    private fun createDialog(
```

Comment:
Please move the function content (dialog logic) into the `onClickListener` below where it's called. We only use it once and it's only a very simple function, so there's no need to have it in a separate function.